src/Domain/Entities/Member.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Codidact.Domain.Common;
+using System;
+
+namespace Codidact.Domain.Entities
+{
+    /// <summary>
+    /// An Entity that represents a member of the community
+    /// </summary>
+    public class Member : AuditableEntity
+    {
+        /// <summary>
+        /// Auto Incremented Identification number
+        /// </summary>
+        public long Id { get; set; }
+        
+        /// <summary>
+        /// The display name that is to be displayed for the member
+        /// </summary>
+        public string DisplayName { get; set; }
+
+        /// <summary>
+        /// The bio information of the member
+        /// </summary>
+        public string Bio { get; set; }
+
+        /// <summary>
+        /// The main email of the member
+        /// </summary>
+        public string Email { get; set; }
+
+        /// <summary>
+        /// Geographical location of the member
+        /// </summary>
+        public string Location { get; set; }
+
+        /// <summary>
+        /// Whether the member is imported from StackExchange or not
+        /// </summary>
+        public bool IsFromStackExchange { get; set; }
+
+        /// <summary>
+        /// The StackExchange Id if it was imported
+        /// </summary>
+        public long? StackExchangeId { get; set; }
+
+        /// <summary>
+        /// When was the data validated against StackExchange
+        /// </summary>
+        public DateTime? StackExchangeValidated { get; set; }
```

Comment:
Missing an `At` off the end of this since it's a datetime field: `StackExchangeValidatedAt`

Reply:
I took these names from the DB Schema threads so I have a few questions: 
Do you mean to have this At only in the code and not the database?
Also, the common columns should they also be CreateDateAt instead of CreateDate?

Reply:
Hm. This may have got missed in the multiple rounds of schema feedback. Some of the feedback was that we should name date fields consistently, ending them either in `...Date` or `...At`. I don't recall what the conclusion was; I suggested At because it seems to make more sense to me. That would make these fields `StackExchangeValidatedAt`, `StackExchangeLastImportedAt`, `CreatedAt`, `ModifiedAt`, etc - so they all make grammatical sense. That'd be both in the database and in the code.

Reply:
Fixed in 0f2a383ea5b7923fee63d169354f62c836749556

Reply:
@ArtOfCode- That sounds grammatically awkward.  "At" refers to a place; not a time.  Nobody says something will happen at Monday; they say it will happen *on* Monday.

Reply:
If we were talking about _dates_, @masonwheeler, yes - it would be *on* 2020-01-17. However, we're talking about date*times*, in which case it's *at* 22:00 2020-01-17.

Reply:
Good point. Objection withdrawn.